browser/extensions/api/brave_rewards_api.cc

```diff
@@ -34,6 +34,15 @@ using brave_rewards::RewardsServiceFactory;
 namespace extensions {
 namespace api {
 
+namespace {
```

Comment:
You do not need to change anything here as it will not be used in the panel

Reply:
now that I have had to rebase again I will test publisher ads are working as expected then remove the UI and put PR up for review, thanks